pkg/operation/botanist/component/etcd/bootstrap.go

```diff
@@ -112,31 +113,46 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 					Resources: []string{"events"},
 					Verbs:     []string{"create", "get", "list", "watch", "patch", "update"},
 				},
+				{
+					APIGroups: []string{corev1.GroupName},
+					Resources: []string{"serviceaccounts"},
+					Verbs:     []string{"get", "list", "watch", "create", "update", "patch", "delete"},
+				},
+				{
+					APIGroups: []string{rbacv1.GroupName},
+					Resources: []string{"roles", "rolebindings"},
+					Verbs:     []string{"get", "list", "watch", "create", "update", "patch", "delete"},
+				},
 				{
 					APIGroups: []string{corev1.GroupName, appsv1.GroupName},
 					Resources: []string{"services", "configmaps", "statefulsets"},
 					Verbs:     []string{"get", "list", "patch", "update", "watch", "create", "delete"},
 				},
+				{
+					APIGroups: []string{batchv1.GroupName},
+					Resources: []string{"jobs", "cronjobs"},
```

Comment:
Doesn't cronjobs come from `batchv1beta1` that I earlier mentioned?

Reply:
The group name is the same for both. Still, I separated them as suggested, I agree it's cleaner this way.